obal/__main__.py

```diff
@@ -1,5 +1,5 @@
 """
 Main module file. This allows you to run python -m obal
 """
-from obal import main
+from obal import main  # pylint:disable=no-name-in-module
```

Comment:
Odd that I didn't have to do this. How did you install your env?

Reply:
I was using `tox -e py27`